index.js

```diff
@@ -80,7 +111,7 @@ function lint(source, options, webpack, done) {
         emitter = webpack.emitWarning;
       }
 
-      emitter(options.formatter(report));
+      emitter(new Error(options.formatter(report)));
```

Comment:
On my dev machine all the tests are green. And I added this because of a warning in friendly-errors-webpack-plugin : "value string instead of error instance".
I do not understand why they are failing on Travis :/